docs/taqo_install_instructions.md

```diff
@@ -0,0 +1,137 @@
+     ![alt_text](images/2-auth-1.png "image_tooltip")
+
+
+5. Once you authenticate, the browser will tell you that you can close the window. 
+
+    ![alt_text](images/3-auth-2.png "image_tooltip")
+
+
+6. You can return to the Taqo app and it will search for your research study.
+    
+    ![alt_text](images/4-find.png "image_tooltip")
+
+
+7. Select the study by clicking on its title. Press the button in the lower right corner to enroll. 
+    
+    ![alt_text](images/5-details.png "image_tooltip")
+
+
+8. You will be shown the Informed Consent and Data Handling agreement, only proceed if you understand and accept these terms.
+
+     ![alt_text](images/6-content.png "image_tooltip")
+
+
+9. Then you will be shown the Schedule for sampling. You can modify this schedule to suit your work hours. Click the save button in the lower right once you are done and use the back arrow to navigate out.
+
+    ![alt_text](images/7-schedule.png "image_tooltip")
+    You are now enrolled in the study and can close the Taqo app. It will notify you if it is time to participate in a survey.
+
+
+    ![alt_text](images/8-joined.png "image_tooltip")
+
+
+10. NOTE: If you are participating in a developer logging study, please do the following two steps to enable the loggers. Otherwise, the study will not collect developer activity and may not be able to detect events that would trigger surveys.
+
+11. Restart any IntelliJ-based IDEs to enable the dev logging plugin
+
+
+12. Restart your terminal or source your .bashrc or .zshrc file to enable terminal command logging
+     `%> source ~/.bashrc`
+
+
+### Responding to a Survey Notification {#responding-to-a-survey-notification}
+
+If the study needs you to fill out a survey, it will pop up a desktop notification. Click that notification to be taken to the survey. Or, you can open Taqo and click on the Study name to fill out the survey.
+
+
+![alt_text](images/9-running.png "image_tooltip")
+
+Click Submit in the lower right hand corner when you are done.
+
+![alt_text](images/10-survey.png "image_tooltip")
+
+
+### Pausing the study {#pausing-the-study}
+
+To pause data collection, open Taqo and click the “⏸” icon to the right of the study name. This will pause the survey’s data collection until you click again to resume.
+
+![alt_text](images/11-pause.png "image_tooltip")
+
+
+### Leaving the study {#leaving-the-study}
+
+At any time, you can open the Taqo app and stop the study by clicking the ‘X’ icon to the right of the study name. 
+
+Note: If you are in a logging study, please leave the study by this method to cleanly remove any IDE plugins and shell loggers (located in the .rc file) from your desktop environment.
+
+![alt_text](images/12-leave.png "image_tooltip")
+
+After leaving a study explicitly, or, if a study has ended, you can uninstall Taqo. Please stop the taqo\_daemon first.
```

Comment:
Do we need to mention that one can run
```
taqo_daemon stop
```
to stop the daemon? Or do we simply remove this since the uninstall script will stop the daemon?

Reply:
I think we can skip that since the script does it.